File: SEM1/C Programs/sample program set1/triangle area.c
//sample6 area of triangle
#include<stdio.h>
#include<conio.h>
main()
{
	int b,h,a;
	printf("enter base of triangle ");
	scanf("%d" ,&b);
	printf("enter height of triangle ");
	scanf("%d" ,&h);
	a = 0.5*b*h;
	printf("Area of triangle is = %d",a);
}

File: SEM3/JAVA/3BCA/Lab_Practice_Programs/triangle_area.java
import java.util.*;
public class triangle_area
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        float h,b,a;
        System.out.print("\n\nTHIS PROGRAM PRINTS AREA OF TRIANGLE\n");
        System.out.print("Enter Height of Triangle : ");
        h=sc.nextFloat();
        System.out.print("Enter Base of Triangle : ");
        b=sc.nextFloat();
        a=0.5f*h*b;
        System.out.print("Area of Triangle : "+a);
    }
}